src/tags/core.js

```diff
@@ -17,6 +17,16 @@ function intStringify(node, radix, prefix) {
   return stringifyNumber(node)
 }
 
+function stringifyBool(node) {
+  const { value } = node
+  let { sourceStr } = node
+  if (sourceStr) {
+    const match = sourceStr.match(/^(?:[Tt]rue|TRUE|[Ff]alse|FALSE)$/)
```

Comment:
1. There should not be a reason to repeat the RegExp here; it's already available as `nullObj.test`, and that way there's no danger of this becoming unconnected if the user modifies the tag.
2. If you're not using the result, use `re.test(str)` instead.

Reply:
done